modules/shared/partials/acid-transactions.adoc

```diff
@@ -87,21 +96,24 @@ Each such ATR entry stores some metadata and, crucially, whether the attempt has
 In this way, the entry acts as the single point of truth for the transaction, which is essential for providing an 'atomic commit' during reads.
 
 === Staged Mutations
+
 The second mechanic is that mutating a document inside a transaction, does not directly change the body of the document.
 Instead, the post-transaction version of the document is staged alongside the document (technically in its xref:concept-docs:xattr.adoc[extended attributes (XATTRs)]).
 In this way, all changes are invisible to all parts of the Couchbase Data Platform until the commit point is reached.
 
 These staged document changes effectively act as a lock against other transactions trying to modify the document, preventing write-write conflicts.
 
 === Cleanup
+
 There are safety mechanisms to ensure that leftover staged changes from a failed transaction cannot block live transactions indefinitely.
-These include an asynchronous cleanup process that is started with the creation of the `Transactions` object, and scans for expired transactions created by any application, on all buckets.
+These include an asynchronous cleanup process that is started with the creation of the transaction, and scans for expired transactions created by any application, on all buckets.
```

Comment:
Hmm this is a bit awkward to abstract over.  In SDK integrated world it is started with the first transaction.  But also the cleanup algo is different - it only looks at collections that we've had a transaction on.
In the transaction library world cleanup is not started at the point of a transaction, it is started at the `Transactions` creation point.  And it scans all buckets.
We perhaps need 2 generic snippets here to cover those cases.

Reply:
Ah gotcha, this is quite helpful! 
I think it's a good idea to have two versions of this then, will update 👍 .